docs/source/tutorials/improving_ml_performance.ipynb

```diff
@@ -0,0 +1,3573 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f361b775-0fb3-4f09-89b9-6672775382bb",
+   "metadata": {},
+   "source": [
+    "# Improving ML Performance Via Data Curation with Train vs Test Splits\n",
+    "\n",
+    "In typical Machine Learning projects, we split our dataset into **training** data for fitting models and **test** (or **validation**) data to evaluate model performance. For noisy real-world datasets, detecting/correcting errors in the training data is important to train robust models, but it's less recognized that the test set can also be noisy.\n",
+    "For accurate model evaluation, it is vital to **find and fix issues in the test data** as well. Some evaluation metrics are particularly sensitive to outliers and noisy labels.\n",
+    "This tutorial demonstrates a way to use `cleanlab` (via `Datalab`) to curate both your training and test data, ensuring **robust model training** and **reliable performance evaluation**.\n",
+    "\n",
+    "Here's how we recommend handling noisy training and test data (this tutorial walks through these steps):\n",
+    "\n",
+    "1. Preprocess your training and test data. Use cleanlab to check for issues in the merged dataset like train/test leakage or drift.\n",
+    "2. Fit your ML model to your noisy training data and get its predictions/embeddings for your test data. Use these model outputs with cleanlab to detect issues in your **test** data.\n",
```

Comment:
Added commit here that hopefully clarifies: https://github.com/cleanlab/cleanlab/pull/1071/commits/4a32bd9fe42ead1a047f8e4c251f144e1e7c5e4f

By preprocess we mean:  Do the non-cleanlab data prep work you'd normally do in your ML project.  Eg. normalize continuous values, encode categoricals, handle missing vals, etc.

Reply:
the reason we include this statement is because the cleanlab package is only intended for data that is in ML-suitable format. cleanlab depends on you first training an initial ML model, so your data should be properly formatted for that